app/build.gradle

```diff
@@ -81,7 +81,6 @@ android {
         }
         alpha.initWith(release)
         alpha {
-            debuggable true
```

Comment:
Play publisher does not create task and does not work if the type is debuggable :(